src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let partner;
+
+  if (isMarried) {
+    partner =
+      sex === 'm'
+        ? `${partnerName} is my wife`
+        : `${partnerName} is my husband`;
+  } else {
+    partner = 'I am not married';
+  }
```

Comment:
Better solution is to create variable for partner status - 'wife' or 'husband'. And then use this variable in marriage status message variable